public_html/wp-content/plugins/wc-post-types/inc/rest-api.php

```diff
@@ -159,10 +159,12 @@ function register_additional_rest_fields() {
 function get_avatar_urls_from_username_email( $post ) {
 	$post        = (object) $post;
 	$avatar_urls = [];
+	$email = get_post_meta( $post->id, '_wcb_speaker_email', true );
+	$user_id = get_post_meta( $post->id, '_wcpt_user_id', true );
 
-	if ( $email = get_post_meta( $post->id, '_wcb_speaker_email', true ) ) {
+	if ( $email ) {
 		$avatar_urls = rest_get_avatar_urls( $email );
-	} elseif ( $user_id = get_post_meta( $post->id, '_wcpt_user_id', true ) ) {
+	} elseif ( $user_id ) {
```

Comment:
- [ ] Test if we can fetch avatar URLs properly. 
- [ ] Align variables on line 162 and 163